### PR TITLE
Updated Package Dependencies & Resolved Deprecations

### PR DESCRIPTION
**Pull request by sk5268**
##  Replaced unnecessary import with appropriate class

`1`. in imagebind/data.py
- Instead of importing the entire module, only the `UniformTemporalSubsample` class was imported, as it was the only class needed.

## Replaced deprecated function:
   `2`. in imagebind/data.py:
- Changed `torchvision.transforms._transforms_video.NormalizeVideo` to `torchvision.transforms.transforms.Normalize`.

`3`. In imagebind/models/transformers.py file:
-  Changed `timm.models.layers` to `timm.layers`
 
 `4`. In imagebind/models/multimodal_preprocessors.py file:
 -  Changed `timm.models.layers` to `timm.layers`.

### CODE REVIEW

**Review by sk5268 — COMMENTED** [review]
Works out of the box with latest versions of package in requirements,txt
No specific version of packages necessary.

**Comment by sk5268** [comment]
@rohitgirdhar 
@ChrisjanWust

**Comment by sk5268** [comment]
@ChrisjanWust 
@rohitgirdhar
@msaroufim 
@aelnouby

File: imagebind/data.py [hunk 1]
```diff
@@ -13,11 +13,10 @@
 import torch.nn as nn
 import torchaudio
 from PIL import Image
-from pytorchvideo import transforms as pv_transforms
 from pytorchvideo.data.clip_sampling import ConstantClipsPerVideoSampler
 from pytorchvideo.data.encoded_video import EncodedVideo
 from torchvision import transforms
-from torchvision.transforms._transforms_video import NormalizeVideo
+from torchvision.transforms.transforms import Normalize
```
Comment (ChrisjanWust) [inline imagebind/data.py:19]: These imported names will now raise a NameError further down. Please test with video.

**Review by ChrisjanWust — CHANGES_REQUESTED** [review]

**Comment by ChrisjanWust** [comment]
Looks like these will break video input because of name errors. Please test with video next time.
